README.md

```diff
@@ -70,10 +70,7 @@ python3 manage.py custom_db_sync /tmp/writer.csv
 ##### Run the django server
 
 ```
+EXPORT DJANGO_SETTINGS_MODULE="course_gather.settings.dev_settings"
+EXPORT SECRET_KEY="YOURSECRETKEY"
```

Comment:
What should `SECRET_KEY` be? :thinking:

Reply:
Literally anything within those quotes to make a string. It could be the national anthem for all Django cares.